ompi/mca/topo/treematch/treematch/IntConstantInitializedVector.c

```diff
@@ -2,7 +2,7 @@
 #include <stdio.h>
 #include "IntConstantInitializedVector.h"
 
-int intCIV_isInitialized(int_CIVector * v, int i)
```

Comment:
As indicated by @rhc54 this is imported code that should mostly be left as is. How is this symbol even spilling out ? It seems to be used only in this file and it is not declared as extern in the header !?